azure-configurationservice/azure/configuration/_generated/_azure_configuration_client_imp.py

```diff
@@ -0,0 +1,38 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from msrest.service_client import SDKClient
+from msrest import Serializer, Deserializer
+
+from ._configuration import AzureConfigurationClientImpConfiguration
+from .operations import AzureConfigurationClientImpOperationsMixin
+from . import models
+
+
+class AzureConfigurationClientImp(AzureConfigurationClientImpOperationsMixin, SDKClient):
+    """Represents an azconfig client
+
+    :ivar config: Configuration for client.
+    :vartype config: AzureConfigurationClientImpConfiguration
+
+    :param credentials: Credentials needed for the client to connect to Azure.
+    :type credentials: :mod:`A msrestazure Credentials
```

Comment:
I apologize here for the lack of understanding of msrest :|

From the comment, I understand that the way to authenticate with the service is not by using the connection string, but using active directory? is this true? how does that work?

Reply:
Today Xiang also mentioned to me about the AAD thing. We're thinking of using AAD. Now we use connection string. Let's talk next week about it.